cli/src/cmd/build.rs

```diff
@@ -242,9 +242,6 @@ mod tests {
 
     #[cfg(feature = "test-ci-only")]
     #[test]
-    // FIXME: https://github.com/paritytech/ink/issues/202
-    // currently fails on CI because of global RUSTFLAGS overriding required `--import-memory`
-    #[ignore]
```

Comment:
Why remove this comment? Is it outdated?

Reply:
yes, fixed in #216